src/frontend/web_application/src/components/ContactDetails/components/AddressForm/index.jsx

```diff
@@ -1,5 +1,6 @@
 import React, { Component } from 'react';
 import PropTypes from 'prop-types';
+import { CountryDropdown, RegionDropdown } from 'react-country-region-selector';
```

Comment:
`react-country-region-selector` is ~83kB minified, it's good candiate for both async-loading and some webpack-magic